ui/yarn.lock

```diff
@@ -1824,7 +1824,7 @@ are-we-there-yet@~1.1.2:
 
 "argo-ui@https://github.com/argoproj/argo-ui.git":
   version "1.0.0"
-  resolved "https://github.com/argoproj/argo-ui.git#3d2740a5531f77e137ea6470314a03a0fdf5f937"
+  resolved "https://github.com/argoproj/argo-ui.git#4e9c9c96ded27da0f008b0ef17c73b418d34118f"
```

Comment:
@keithchong please re-run `yarn add https://github.com/argoproj/argo-ui.git` to pull most recent argo-ui changes. This should make CI green

Reply:
Hi @alexmt , I think https://github.com/argoproj/argo-ui/pull/76 needs to go in first, otherwise argo-cd will not build cleanly.  See https://github.com/argoproj/argo-cd/issues/4913. 

I think this PR could wait until 76 and the recent type-refactoring changes are resolved.

Reply:
Merged argo-ui#76. Thank you for your help with reviewing and testing it!

Reply:
Np @Alexmt.  There is still one more thing to do before this can be merged, because this will pull in Daniel's argo-ui changes too. In https://github.com/argoproj/argo-cd/issues/4913 he said that argo-cd will build with his change but argo-cd needs to have its `react` and `react-dom` version changed to at least 16.9.3.